R/g_lineplot.R

```diff
@@ -29,6 +29,7 @@
 #' Default value is `ggplot2::waiver()`.
 #' @param rotate_xlab boolean whether to rotate x-axis labels.
 #' @param plot_font_size control font size for title, x-axis, y-axis and legend font.
+#' @param dot_size plot dot size.
```

Comment:
Since it is the `size` argument of `geom_point` I think `point_size` would be better. WDYT?
We can also mention here that this is the size of the plot's point along with the default of 3.
Also, please add this info to the news, that the line and spaghetti plot now supports custom plot size.

Reply:
`g_correlationplot` has this parameter with name `dot_size,` I kept the same name just for consistency. but I have seen some inconsistent parameter name e.g `font_size` and` plot_font_size`. 

should I just update name for these two functions. making change in correlation may cause issues .

Reply:
I see, in that case, I think it's best to just stick to `dot_size` instead of deprecating the name in other functions now. Perhaps the depreciation can be done in the future.

Reply:
WDYT about adding this param to the examples as well. I see that the examples already have other optional params too.

Reply:
sounds good.  done [1887a7e](https://github.com/insightsengineering/goshawk/pull/242/commits/1887a7ee00246a2c499b1d993ff16123ba73c3a4)